### PR TITLE
docs(tekton): add readme file in core and pipelines package

### PR DESCRIPTION
**Pull request by karthikjeeyar**
Add Readme in core and pipelines package

### CODE REVIEW

File: packages/pipelines/package.json [hunk 1]
```diff
@@ -33,17 +33,19 @@
     "url": "https://github.com/redhat-developer/aonic-ui/issues"
   },
   "homepage": "https://github.com/redhat-developer/aonic-ui#readme",
+  "dependencies": {
+    "@patternfly/react-core": "^5.1.2",
+    "@patternfly/react-icons": "^5.1.2",
+    "@patternfly/react-table": "^5.1.2",
+    "@patternfly/react-tokens": "^5.1.2"
```
Comment (rohitkrai03) [inline packages/pipelines/package.json]: Shouldn't these come from peer dependencies?

Comment (rohitkrai03) [reply packages/pipelines/package.json]: Also, we don't need `"@testing-library/jest-dom": "^6.2.0"` in peer dependencies.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: packages/pipelines/package.json (hunk 1 above)
Comment (karthikjeeyar) [reply packages/pipelines/package.json]: Updated.

File: packages/pipelines/package.json [hunk 2]
```diff
@@ -52,12 +54,7 @@
     "typescript": "^5.3.3"
   },
   "peerDependencies": {
-    "react": "^18.2.0",
-    "@testing-library/jest-dom": "^6.2.0",
-    "@patternfly/react-core": "^5.1.2",
-    "@patternfly/react-icons": "^5.1.2",
-    "@patternfly/react-table": "^5.1.2",
-    "@patternfly/react-tokens": "^5.1.2"
```
Comment (rohitkrai03) [inline packages/pipelines/package.json]: These should still be in peerDependencies as the main application might bring in these as well.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]
Do we still need yarn.lock changes?

**Comment by karthikjeeyar** [comment]
removed yarn.lock

**Review by rohitkrai03 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**